app/templates/components/dwd-rainradar.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
you can delete this file. Ember's default template for components is sufficient for this :)

Reply:
File deleted